flask_and_kafka/formatters.py

```diff
@@ -22,6 +22,9 @@ class ProducerFormatter(logging.Formatter):
 
     def format(self, record):
         super().format(record)
+        for entry_name, entry_value in record.producer_log.items():
```

Comment:
Logically there is no reason for us to convert all the inputs because the other side might need a byte!! So the best thing is to do the following:
```
def convert_bytes_to_repr(value) -> str :
    if isinstance(value, bytes):
        return repr(value)
    return value

'key': convert_bytes_to_repr(key),
'value': convert_bytes_to_repr(value),
```